engine/flink/schemed-kafka-components-utils/src/test/scala/pl/touk/nussknacker/engine/schemedkafka/source/flink/TestWithTestDataSpec.scala

```diff
@@ -89,6 +89,26 @@ class TestWithTestDataSpec extends AnyFunSuite with Matchers with LazyLogging {
 
   }
 
+  test("should handle fragment test parameters in test") {
```

Comment:
We have to add tests also using our `TestScenarioRunner` mechanism - for both Flink and Lite

Reply:
Doesn't `TestScenarioRunner` test scenarios invocation? We are never running fragment "as scenario" - we can only run fragment by test mechanism. There is no scenario in which we can call fragment as we would do with scenario. I might be wrong but isn't it the case? :thinking:

Reply:
Thanks to `TestScenarioRunner` our users can test various scenario constructions in unit test. I think, that they also should be able to tests fragments as a decomposed problem that will be composed in scenario. I know that it can look a little bit "out of scope" but  IMO thanks to that we will see if this implementation is good or not. So far everything that is possible in test in GUI mechanism is also possible in our "testkit" framework

Reply:
The idea is great but it sounds like some refactor is needed to handle fragments in such a form. I wonder if that isn't something we should do separately from this MR?

Reply:
It might be in separate MR